tests/test_aggregator.py

```diff
@@ -920,7 +920,7 @@ def assert_send_runs_correctly(self, method, url, data=None, headers=None, to_po
     If any of the SEND attempts succeeds then the routine exits immediately succesfully.
 """
         aggregator_urls_queue = [ x for x in aggregator_urls ]
-        def _get_href(_):
+        def _get_href(_1, _2, _3, _4):
```

Comment:
Why does this function have arguments? They don't appear to be used anywhere

Reply:
It's simply to match the signature of what it's mocking/capturing. We only need to know it 'has been called'.